README.md

```diff
@@ -10,32 +13,23 @@
 - Robert Tibshirani
 
 - Daniela Witten
-  
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/intro-stat-learning/ISLP_labs/v2.2)
 
 ### ISLP
 
-Please ensure you have followed the installation instructions for
-[ISLP](https://github.com/intro-stat-learning/ISLP). This will address
-installation of [jupyterlab](https://github.com/jupyterlab/jupyterlab)
-if necessary, which is not included as a requirement of the labs.
-
-### Up-to-date version of labs for ISLP. 
-
 This repo will track labs for ISLP as their source code changes.  The
-intent is that building a conda environment with
+intent is that building a virtual environment with
 `requirements.txt` will reproduce the results in this repo.
 
 To install the current version of the requirements run
 
 ```
-pip install -r https://raw.githubusercontent.com/intro-stat-learning/ISLP_labs/v2.2/requirements.txt;
```

Comment:
One worry about `make install` here is for Windows users who may not use CLI. The pip install can be run within a notebook as a magic where I don't know about the calls to `make`. Won't users have to have downloaded Apple's CLI tools for this?

Reply:
This doesn't mean we can't have the `Makefile` of course, just perhaps the instructions for students shouldn't presume `make`. That's all.